docs/index.rst

```diff
@@ -25,7 +25,7 @@ A great way of explaining the library is using an example notebook, which you ca
 
 The most recent version of this library adds a feature to leverage unlabelled images in order to improve the performance
 of image classifiers. This procedure is called semi-supervised learning (SSL) and is discussed in this `blog post <https://medium.com/decathlondevelopers/improving-performance-of-image-classification-models-using-pretraining-and-a-combination-of-e271c96808d2/>`_.
-The method was also described in a `paper <https://arxiv.org/abs/2108.08362/>`_ and presented at a conference.
+The method was also described in a `paper <https://arxiv.org/abs/2108.08362/>`_ and presented at a `conference <http://mmsports.multimedia-computing.de/mmsports2021/program.html>`_.
```

Comment:
Maybe we can add:

..presented at ACM MMSports 2021

Reply:
Good idea. I included your suggestions.